alethzero/NatspecHandler.h

```diff
@@ -26,7 +26,8 @@
 #pragma warning(disable: 4100 4267)
 #include <leveldb/db.h>
 #pragma warning(pop)
-#include <json/json.h>
+//#include <json/json.h>
+#include <jsoncpp/json/json.h>
```

Comment:
please don't do that, it may break the build for others. #1038 should fix all issues with it. Please revert "#include <jsoncpp/json/json.h>"  and rebase to latest develop. Cleaning build directory may also be required.